public/app/features/dimensions/editors/FileUploader.tsx

```diff
@@ -0,0 +1,91 @@
+import React, { Dispatch, SetStateAction } from 'react';
+import { FileDropzone, useTheme2, Icon, Button } from '@grafana/ui';
+import { getBackendSrv, config } from '@grafana/runtime';
+import { GrafanaTheme2 } from '@grafana/data';
+import { css } from '@emotion/css';
+interface Props {
+  setNewValue: Dispatch<SetStateAction<string>>;
+}
+
```

Comment:
I think icon panel only supports SVGs - should this component receive a list of acceptable extensions/file formats to prevent the user from selecting unsupported files?